test/runtests.jl

```diff
@@ -2,7 +2,6 @@
 
 using Aqua
 using Documenter
-using ForwardDiffChainRules
```

Comment:
why?

Reply:
Because I ended up writing the ForwardDiff compatibility myself, so it wasn't needed anymore. It annoyed me that the users had to import ForwardDiffChainRules and use a macro, I wanted the package to be as self-contained as possible.